llama_stack/cli/stack/configure.py

```diff
@@ -145,13 +146,17 @@ def _configure_llama_distribution(
                 "yellow",
                 attrs=["bold"],
             )
-            config = StackRunConfig(**yaml.safe_load(run_config_file.read_text()))
+            config_dict = yaml.safe_load(run_config_file.read_text())
+            config = parse_and_maybe_upgrade_config(config_dict)
         else:
             config = StackRunConfig(
                 built_at=datetime.now(),
                 image_name=image_name,
-                apis_to_serve=[],
-                api_providers={},
+                apis=list(build_config.distribution_spec.providers.keys()),
```

Comment:
apis is not the full list of all apis provided by the distribution itself right? just the ones for which there are providers.

/inspect, /models, /shields, /memory_banks are not part of apis?